compiler/cpp/src/thrift/generate/t_swift_3_generator.cc

```diff
@@ -0,0 +1,2553 @@
+/*
```

Comment:
@nsuke I think I'd actually prefer to eliminate the old Swift generator and rename this to just t_swift_generator.cc, given that Apple will sunset Swift 2.x within the next 6 months.  Is this acceptable?  

My current fear with that is that the current swift lib/generator are not housed under a tag here, so anyone having a dependency may face issues.  
Anyone currently using it is almost certainly using Cocoapods and pointing to master, and they'd have a lockfile pointing to the commit they last used, so there's some certainty there, but an intermediate snapshot tag might help otherwise. 

Cocoapods does support pointing to explicit commits though too, a small How-To could be added to the README to help users in need to point to the old generator if they overly specifically need it.